ComfortZoneApp/app/src/main/java/com/example/comfortzone/callback/UserDetailsCallback.java

```diff
@@ -2,8 +2,12 @@
 
 import com.example.comfortzone.models.WeatherData.Coordinates;
 
-public interface UserLocationCallback {
+import java.util.HashSet;
```

Comment:
@fionamei i have noticed that sometimes you have somewhat long commit messages. Here you could have shortened the commit message as "[passing in saved cities] changed to hashmap instead of arraylist", you dont need to actually specify the reason for the change int he commit message. You can provide more details about why you are implementing the change and provide implementation details int he description rather than the commit message.